apps/coscrad-frontend/src/store/index.ts

```diff
@@ -25,6 +25,8 @@ import { PHOTOGRAPHS } from './slices/resources/photographs/constants';
 import { songReducer, SONGS } from './slices/resources/songs';
```

Comment:
Just curious if this is going to change to match the video directory structure of `./constants`?

Reply:
I should be barrel exporting these slice names from the top level of each resource's slice directory. I can try to clean this up. Great point!